x-pack/plugins/security_solution/public/cases/containers/use_post_case.test.tsx

```diff
@@ -66,7 +64,6 @@ describe('usePostCase', () => {
       result.current.postCase(samplePost);
```

Comment:
I think we can test that the case have been posted correctly.

```
it('post case', async () => {
    await act(async () => {
      const { result, waitForNextUpdate } = renderHook<string, UsePostCase>(() => usePostCase());
      await waitForNextUpdate();
      const postData = await result.current.postCase(samplePost);
      expect(postData).toEqual(basicCasePost);
    });
  });
```

Reply:
The postData I have is a promise, so I wrap it with waitFor and put the test in another block with its own description.

Reply:
Hey, you can always `await result.current.postCase(samplePost)`. Is there any need for `waitFor` ?